dashboards/node-overview.json

```diff
@@ -85,6 +85,23 @@
         }
       }
     },
+    {
+      "title": "sysproc cpu utilization (Trinity+)",
+      "datasource": "{data-source-name}",
+      "_base": "panel",
+      "_targets": [
+        {
+          "expr": "(rate(sysproc_cpu_user[1m]) + ignoring(name) rate(sysproc_cpu_sys[1m])) / ignoring(category, name, proc) group_left sys_cpu_cores_available * 100",
```

Comment:
For unclear reason this formula is not working for me locally (on mac). It shows 200+% load on cluster that is doing nothing. We might have a bug somewhere.